rrrspec-client/lib/rrrspec/client/slave_runner.rb

```diff
@@ -3,6 +3,9 @@
 require 'timeout'
 require 'rspec/core/formatters'
 
+include Cucumber::RbSupport::RbDsl
```

Comment:
I don't think we'd like to introduce the Cucumber dependency to RRRSpec itself. Can you make an abstraction that we can make this optional?